api/db/repository/petition.py

```diff
@@ -62,12 +84,74 @@ def get_by_id(_db: Session, petition_id: int, petition_status=""):
     return db_petition.first()
 
 
-def get_all(_db: Session, offeset: int = 0, limit: int = 100):
-    return _db.query(Petition).offset(offeset).limit(limit).all()
+def get_all(
+    _db: Session,
+    offset: int = 0,
+    limit: int = 100,
+    statuses: List[str] = None,
+    creation_date: List[str] = None,
+    due_date: List[str] = None,
```

Comment:
Why due date is list?

Reply:
Here I pass dates like a list because if we give a single date to the URL, in that case we will filter where the date is equal to passed one, but  if we pass two dates in the list it will filter where dates are in the range